lib/client.js

```diff
@@ -311,7 +347,48 @@ Client.prototype = {
     }.bind(this));
   },
 
+  /// #### client.instance(instanceGuid)
+  ///
+  /// Create an Client wrapper for another app instance.
+  ///
+  /// ##### Arguments
+  ///
+  ///   * `instanceGuid` the GUID of the other instance of your app
+  ///
+  /// ##### Returns
+  ///
+  /// A [Promises/A+](https://promisesaplus.com) conformant `promise` object.
+  ///
+  /// ```javascript
+  /// var client = ZAFClient.init();
+  ///
+  /// client.get('instances').then(function(data) {
+  ///   var guid = Object.keys(data.instances).filter(function(guid) {
+  ///     return data.instances[guid].location == 'top_bar';
+  ///   })[0];
+  ///   return guid;
+  /// }).then(function(guid) {
+  ///   return client.instance(guid);
+  /// }).then(function(topBarClient) {
+  ///   topBarClient.invoke('popover');
+  /// });
+  /// ```
+  instance: function(instanceGuid) {
+    if (this._parent) { return this._parent.instance(instanceGuid); }
+    var instanceContextKey = 'instances.' + instanceGuid;
+    this._instanceClients[instanceGuid] = this._instanceClients[instanceGuid] ||
```

Comment:
fair enough, we'd need an extra level of nesting in the shared object just to keep it private. Carry on